Configuration/Eras/python/ModifierChain_trackingMkFitProd_cff.py

```diff
@@ -16,5 +15,4 @@
     trackingMkFitHighPtTripletStep,
     trackingMkFitDetachedQuadStep,
     trackingMkFitDetachedTripletStep,
-    trackingMkFitPixelLessStep,
```

Comment:
Shall we comment it, hoping to put it back ~ soon?

Reply:
IIRC, comments are usually not advised by reco: if some change is expected in the code soon, the usual advice is to make an issue in github